app/routers/apis.py

```diff
@@ -89,13 +89,25 @@ async def update_runtime_configuration(new_configuration_keys: Dict[str, Any]):
     return PlainTextResponse("OK")
 
 
+@router.get("/runtime/metadata")
```

Comment:
Can you also add this path to https://github.com/paritytech/testnet-manager/blob/main/tests/smoke_test.sh#L19